tests/integration/Owncloud/OcisSdkPhp/OcisTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace integration\Owncloud\OcisPhpSdk;
+
+use GuzzleHttp\Client;
+use Owncloud\OcisPhpSdk\Exception\ForbiddenException;
+use Owncloud\OcisPhpSdk\Ocis;
+use PHPUnit\Framework\TestCase;
+
+class OcisTest extends TestCase
+{
+    private const CLIENT_ID = 'xdXOt13JKxym1B1QcEncf2XDkLAexMBFwiT9j6EfhhHFJhs2KM9jbjTmf8JBXE69';
+    private const CLIENT_SECRET = 'UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh';
+    private const UUID_REGEX_PATTERN = '/^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}\$[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i';
+    private string $ocisUrl = 'https://ocis.sdk.test:9009';
+    private ?string $tokenUrl = null;
+    private ?Client $guzzleClient = null;
+    /**
+     * @var array <string>
+     */
+    private $createdDrives = [];
+
+    public function setUp(): void
```

Comment:
why you name this setUp?

Reply:
This is how the method, that runs before every test, has to  be called in PHPUnit
https://docs.phpunit.de/en/10.4/fixtures.html